src/transformers/models/llama/modeling_llama.py

```diff
@@ -692,7 +692,6 @@ def custom_forward(*inputs):
                     hidden_states,
                     attention_mask,
                     position_ids,
-                    None,
                 )
```

Comment:
should be output_attention.
```suggestion
                    output_attentions,
```